libraries/rush-lib/src/logic/base/BaseInstallManager.ts

```diff
@@ -574,8 +574,16 @@ export abstract class BaseInstallManager {
         args.push('--network-concurrency', options.networkConcurrency.toString());
       }
 
-      if (this._rushConfiguration.pnpmOptions.strictPeerDependencies) {
-        args.push('--strict-peer-dependencies');
+      if (semver.gte(this._rushConfiguration.pnpmVersion, '7.0.0')) {
+        // pnpm >= 7.0.0 handles peer dependencies strict by default
+        if (this._rushConfiguration.pnpmOptions.strictPeerDependencies === false) {
+          args.push('--no-strict-peer-dependencies');
```

Comment:
Update the `rush.schema.json` to specify that the default behaviour is the package manager default. Currently it says that the default behaviour is to set it to `false`.

Reply:
Got it. Will do so. Thanks.

Reply:
`rush.schema.json` updated accordingly